### ISSUE TITLE
Netclient binary in the wrong place - problem with selinux

### ISSUE THREAD

**Issue by a1ad** [gravitl/netmaker#514]
Netclient is not working on Fedora 35 with selinux enforced:

```
netclient[73362]: 2021/11/24 09:08:50 [netclient] error checking in for testNET network: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp 10.30.66.111:443: connect: permission denied"
nov 24 09:08:50 400833016 systemd[1]: netclient.service: Deactivated successfully.

```

The netclient binary is installed in /etc/netclient/netclient en symlinked to  /usr/local/bin/netclient. The systemd service is using the /etc/netclient/netclient  location. But selinux dont like binaries in the wrong place. So when I removed the symlink, copied the binary to /usr/local/bin/.. and adjusted the systemd service to make use of the  /usr/local/bin/netclient binary i v got a working netclient

So we don't need to adjust seliux thankfully, we just need to live by its rules.

As a sidenote. I also tested the use of the symlink in /usr/local/bin, but that is not working either, so you really need to put the binary i the right place

To sum it up:

1: remove symlink /usr/local/bin/netclient
1: copy /etc/netclient/netclient to /usr/local/bin/
2: edit: /etc/systemd/system/netclient.service to make use of the new binary location: /usr/local/bin/netclient
3:  reload the systemd daemon: systemctl daemon-reload
4: restart netclient: systemctl restart netclient 

result:

```
nov 24 09:14:38 400833016 netclient[73994]: 2021/11/24 09:14:38 [netclient] running checkin for all networks
nov 24 09:14:38 400833016 netclient[73994]: 2021/11/24 09:14:38 [netclient] checked in successfully for testNET
nov 24 09:14:39 400833016 netclient[73994]: 2021/11/24 09:14:39 [netclient] checked in successfully for testNET
nov 24 09:14:39 400833016 systemd[1]: netclient.service: Deactivated successfully.
```

**Comment by afeiszli** [comment]
Marking this as an enhancement for the next release. Should be pretty easy to put in place.
